azext_iot/tests/digitaltwins/test_dt_model_unit.py

```diff
@@ -0,0 +1,710 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import pytest
+import responses
+import json
+import urllib
+from knack.cli import CLIError
+from azext_iot.digitaltwins import commands_models as subject
+from azext_iot.tests.generators import generate_generic_id
+from msrest.paging import Paged
+
+instance_name = generate_generic_id()
+hostname = "{}.subdomain.domain".format(instance_name)
+resource_group = 'myrg'
+api_version = "2020-10-31"
+
+
+def generate_model_id():
+    normal_id = "dtmi:com:{}:{};1".format(generate_generic_id(), generate_generic_id())
+    url_id = urllib.parse.quote_plus(normal_id)
+    return normal_id, url_id
+
+
+model_id, url_model_id = generate_model_id()
+generic_result = json.dumps({"result": generate_generic_id()})
+
+
+def generate_model_result(model_id=None):
+    model_id = model_id if model_id else generate_model_id()[0]
+    return {
+        "model": {
+            "@context" : ["dtmi:com:context;2"],
+            "@id" : model_id,
+            "@type" : "Interface"
+        },
+        "id": model_id
+    }
+
+
+@pytest.fixture
+def control_and_data_plane_client(mocker, fixture_cmd):
```

Comment:
Can we call this a fixture? Also seems like there could be re-use across other tests (like the twin unit tests) so we don't have to redefine it.